content/ru/docs/reference/glossary/object.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Объект
+id: object
+date: 2023-11-20
+full_link: /docs/concepts/overview/working-with-objects/#kubernetes-objects
+short_description: >
+   Сущность в системе Kubernetes, представляющая часть состояния кластера.
+aka: 
+tags:
+- fundamental
+---
+Сущность в системе Kubernetes. Kubernetes использует их для представления состояния кластера. 
+<!--more-->
+Объект Kubernetes обычно представляет собой «запись о намерениях» — как только объект создан,
+{{< glossary_tooltip text="слой управления" term_id="control-plane" >}} Kubernetes постоянно работает,
```

Comment:
Doesn't it sound a bit awkward otherwise?
```suggestion
{{< glossary_tooltip text="слой управления" term_id="control-plane" >}} Kubernetes обеспечивает гарантию того,
```

Reply:
Yes, работает does not sound right in this context. Thank you!